matlabMeas/MeasurementUtilities/initializeSR830Meas2D_Func.m

```diff
@@ -1,73 +1,62 @@
-function [plotHandle] = initializeSR830Meas2D_Func(sweepType1, sweepType1,doBackAndForth)
+function [plotHandle] = initializeSR830Meas2D_Func(sweepTypes, starts, stops, deltas)
 
-[time,Real,Imag] = deal(inf);
+% initializeSR830Meas2D_Func({'Freq', 'ST'}, {1000, 0}, {10000, 1}, {1000, 0.01})
```

Comment:
Why is this hard coded? Make this general.

Reply:
I forgot to fully flesh out the comment; it's just something for me to easily copy and paste for debugging. It should work for all cases you've currently defined for sweeping.

In addition, i'd like to make a more generalized case function that deals with doing different sweeps, but that can be done later.

Reply:
A generalized sweep can be done using setVal. That function takes care setting the proper devices and their constituent values.